docs/snippets/angular/my-component-disable-toc.ts.mdx

```diff
@@ -0,0 +1,21 @@
+```ts
+// MyComponent.stories.ts
+
+import type { Meta } from '@storybook/angular';
```

Comment:
We ran into some issues in the past, with some users reporting that the examples weren't being depicted accurately for Angular, and we've adopted this strategy to avoid further confusion. I'm aware that it introduces a maintenance burden, but we're managing it and hopefully in the future this will no longer be required.